client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -197,12 +197,14 @@ const PatternAssembler = ( {
 			pattern_categories: categories.join( ',' ),
 			category_count: categories.length,
 			pattern_count: patterns.length,
+			assembler_source: getAssemblerSource( selectedDesign ),
```

Comment:
I assume we can add `assembler_source` via the [createRecordTrackEvent](https://github.com/Automattic/wp-calypso/pull/78041/files#diff-238ed6a820087c26a46b9cfe70db2c6a0e8e9408913101ee0aad08dddaf5f811R117-R130). WDYT?

Reply:
yes, easier to maintain it from there 👍